mssql-ballerina/tests/batch-execute-query-test.bal

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/sql;
+import ballerina/test;
+
+string batchExecuteDB = "batch_execute_db";
+
+@test:Config {
+    groups: ["batch-execute"]
+}
+function batchInsertIntoExactNumericTable1() {
+    var data = [
+        {row_id: 10, bigintValue: 9223372036854775807, numericValue: 123.34},
+        {row_id: 11, bigintValue: 9223372036854775807, numericValue: 123.34},
+        {row_id: 12, bigintValue: 9223372036854775807, numericValue: 123.34}
+    ];
+    sql:ParameterizedQuery[] sqlQueries =
+        from var row in data
+        select `INSERT INTO ExactNumeric (row_id, bigint_type, numeric_type) VALUES (${row.row_id}, ${row.bigintValue}, ${row.numericValue})`;
+}
+
+@test:Config {
+    groups: ["batch-execute"],
+    dependsOn: [batchInsertIntoExactNumericTable1]
+}
+function batchInsertIntoExactNumericTable2() {
+    int rowId = 15;
+    int intValue = 5;
+    sql:ParameterizedQuery sqlQuery = `INSERT INTO NumericTypes (row_id, int_type) VALUES(${rowId}, ${intValue})`;
+    sql:ParameterizedQuery[] sqlQueries = [sqlQuery];
```

Comment:
This test case looks incomplete. please check

Reply:
Getting an error message like `The statement must be executed before any results can be obtained` for batch executing in MSSQL.
same problem occurs in JDBC driver too as `executeBatch()` method does not support getting generated keys in MSSQL.
https://social.msdn.microsoft.com/Forums/en-US/6cbf5eea-e5b9-4519-8e86-f4b65ce3f8e1/jdbc-batch-insert-getgeneratedkeys?forum=sqldataaccess